handsontable/src/selection/selection.js

```diff
@@ -628,13 +634,16 @@ class Selection {
    * `false` otherwise.
    * @param {boolean} [includeColumnHeaders=false] `true` If the selection should include the column
    * headers, `false` otherwise.
-   * @param {{row: number, col: number}} [focusPosition] The argument allows changing the cell/header
+   * @param {object} [options] Additional object with options.
+   * @param {{row: number, col: number}} [options.focusPosition] The argument allows changing the cell/header
    * focus position. The value takes an object with a `row` and `col` properties from -N to N, where
    * negative values point to the headers and positive values point to the cell range.
+   * @param {boolean} [options.disableHeadersHighlight] If `true`, disables highlighting the headers even when
+   * the logical coordinates points on them.
    */
-  selectAll(includeRowHeaders = false, includeColumnHeaders = false, focusPosition = {
-    row: this.tableProps.countColHeaders() > 0 ? -this.tableProps.countColHeaders() : 0,
-    col: this.tableProps.countRowHeaders() > 0 ? -this.tableProps.countRowHeaders() : 0,
+  selectAll(includeRowHeaders = false, includeColumnHeaders = false, options = {
+    focusPosition: false,
```

Comment:
It seems that `focusPosition` may be a boolean, not described in JSDoc.

Reply:
Done https://github.com/handsontable/handsontable/pull/10464/commits/dd1ddedb5f12be052ed900c581dcf1b1def993f6 👍